kong/db/schema/metaschema.lua

```diff
@@ -72,6 +72,7 @@ local field_schema = {
   { type = { type = "string", one_of = keys(Schema.valid_types), required = true }, },
   { required = { type = "boolean" }, },
   { reference = { type = "string" }, },
+  { description = { type = "string" }, },
```

Comment:
Let's add a minimum and a maximum length limit to these.

Reply:
Added a minimum length of `10` and a maximum length of `500` characters. This tolerates the _existing_ descriptions we have in our docs (with one needed change). The longest description we have is 447 characters long.

As a reference here is an example text with 500 characters:

```
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla efficitur urna neque, eu vehicula libero tristique ac. Vivamus lacinia turpis vel mi feugiat, eu pharetra dui facilisis. Sed in sapien id odio bibendum fermentum nec ut justo. Aenean imperdiet lacinia nisi, nec tempus odio euismod ut. Pellentesque at magna ac ligula commodo aliquet eget a arcu. Proin vitae dolor vel mi vulputate condimentum. Suspendisse bibendum purus velit, ut condimentum odio finibus nec. Fusce scelerisque, quam ut sollicitudin fringilla, sapien mauris tristique odio, vel dictum lectus libero id ante. Integer ullamcorper malesuada sapien, non gravida sapien lobortis vel. Etiam euismod, lorem nec cursus hendrerit, lectus nulla pharetra augue, id pulvinar dui lacus eu turpis. In hac habitasse platea dictumst. Ut sit amet tortor nunc.
```

And an example text with 10 characters.

```
Lorem ipsum d
```

Reply:
Looks good.